src/main/java/com/contrastsecurity/http/RequestConstants.java

```diff
@@ -3,6 +3,8 @@
 public class RequestConstants {
     public static final String AUTHORIZATION = "Authorization";
     public static final String API_KEY = "API-Key";
+    public static final String X_TELEMETRY_INTEGRATION_NAME = "X_TELEMETRY_INTEGRATION_NAME";
+    public static final String X_TELEMETRY_VERSION = "X_TELEMETRY_VERSION";
```

Comment:
Lets make this `X_TELEMETRY_INTEGRATION_VERSION` to be consistent. or make the other `X_TELEMETRY_NAME`

Reply:
recreating a review, forgot to mark for request changes.